CHANGELOG.md

```diff
@@ -6,12 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+* Introduced a thread worker pool for parallel downloads
+
 ### Added
 * Add support for unverified checksum upload mode
 
 ### Fixed
 * Fix downloading files with unverified checksum
 
+
```

Comment:
This empty line should not be here, but the changelog linter should have told you that. I think it's only run in CI while it should also be run in `nox`. Please file a separate issue (or PR) for it.